kymatio/scattering2d/tests/test_utils_scattering2d.py

```diff
@@ -0,0 +1,19 @@
+import pytest
```

Comment:
Since this is testing the generic `Scattering2D` class, why not rename to `test_scattering2d.py`?

Reply:
Alright! Agreed.

Reply:
Ok. Ended up going with `test_frontend_scattering2d.py`.